packages/api/portfolioDrafts/funding/createFundingStep.ts

```diff
@@ -35,7 +36,9 @@ export interface ClinValidationError {
  * @returns a collection of clin validation errors
  */
 export function validateFundingStepClins(fs: FundingStep): Array<ClinValidationError> {
-  return fs.clins.map(validateClin).reduce((accumulator, validationErrors) => accumulator.concat(validationErrors), []);
+  const clins: Array<Clin> = [];
+  fs.task_orders.forEach((taskOrder) => taskOrder.clins.forEach((clin) => clins.push(clin)));
+  return clins.map(validateClin).reduce((accumulator, validationErrors) => accumulator.concat(validationErrors), []);
```

Comment:
So then this should all be able to be:

```typescript
return fs
         .task_orders.flatMap(taskOrder) => taskOrder.clins)
         .map(validateClin)
         .reduce(accumulator, validationErrors) => accumulator.concat(validationErrors), []);
```

+/- some indentation and line-breaks there to appease the linter. In general if you're noticing a need to nest a `forEach` to create a list of elements of inner lists, that can become a `flatMap`.

Reply:
Thanks. Noted and much cleaner.

Reply:
Neat.
> Equivalent to a .map() and a .flat() of depth 1.